packages/daisyui/src/templates/ArrayFieldTitleTemplate/ArrayFieldTitleTemplate.tsx

```diff
@@ -0,0 +1,12 @@
+import { ArrayFieldTitleProps, StrictRJSFSchema, RJSFSchema, FormContextType } from '@rjsf/utils';
+
+export default function ArrayFieldTitleTemplate<
+  T = any,
+  S extends StrictRJSFSchema = RJSFSchema,
+  F extends FormContextType = any
+>(props: ArrayFieldTitleProps<T, S, F>) {
+  console.log('DaisyUI ArrayFieldTitleTemplate');
+
+  props;
+  return <div>ArrayFieldTitleTemplate</div>;
```

Comment:
I'm curious why you implemented this since it does nothing to display the `TitleFieldTemplate`